translations/fr.php

```diff
@@ -190,6 +190,7 @@
 $_MODULE['<{paypal}prestashop>statusblock_b0fb9d7e6d4ed47cb12f3092943fafc2'] = 'Le protocole SSL doit être activé sur votre site.';
 $_MODULE['<{paypal}prestashop>statusblock_ba54b8645ec3d87e95b4da1b5baad66f'] = 'L\'extension cURL PHP doit être activée sur votre serveur.';
 $_MODULE['<{paypal}prestashop>statusblock_5914be43ea057f2fd9fa49180f5db5bf'] = 'Vous devez vous connecter à votre compte PayPal.';
+$_MODULE['<{paypal}prestashop>statusblock_85e650a9b75ce606496731a086c28d0b'] = 'Vous avez des fichiers dans votre module PayPal qui ne sont pas présentes dans le contenu officiel. Voir plus de détails sur [a @href1@]page de diagnostic.[/a]';
```

Comment:
Hello @bogdan202 

A little translation mistake here, please update to : 
"Vous avez des fichiers dans votre module PayPal qui ne sont pas présents"

My bad i saw on the original ticket.

Thank you